modules/reactor/src/meshgenerators/FlexiblePatternGenerator.C

```diff
@@ -151,6 +171,8 @@ FlexiblePatternGenerator::validParams()
       "boundary_type boundary_mesh boundary_sectors boundary_size "
       "delete_default_external_boundary_from_inputs external_boundary_id external_boundary_name",
       "Boundary");
+  params.addParamNamesToGroup("cell_id_name cell_id_shift pattern_id_name pattern_id_shift",
+                              "Reporting Id Parameters");
```

Comment:
```suggestion
                              "Reporting Id");
```
Parameters gets added automatically at the end already